website/docs/ui_fulltextsearch.md

```diff
@@ -8,13 +8,14 @@ Type the desired word in "Search line" to search.
 
 Search modes
 
-* "Default" — This follows the [xapian search syntax](https://xapian.org/docs/queryparser.html). Note that phrase searching and NEAR operator needs `Enable index with positional information` enabled at settings.
+* "Default" — This follows the [xapian search syntax](https://xapian.org/docs/queryparser.html). 
+Note that phrase and NEAR search needs the positional information when indexing.  The positional information will make the xapian index file too big.  The program has used a balanced way to enable or disable the positional information automatically based on the total document length.which means the positional information will not be disabled when the ditionary has reached the limit.
 * "Plain text" - mode like "Whole words" but every word in search line can be treated as word fragment.
-* "Wildcards" - the search line contains a Unix-like template. Such template can contain wildcard symbols `?` (matches any one character), `*` (matches any character number) or ranges of characters `[...]` To find characters `?`, `*`, `[` and `]` it should be escaped by backslash like `\?`, `\*`, `\[`, `\]`.
+* "Wildcards" - As xapian index only support wildcard syntax like this  "hell*" ,  the wildcard in the middle(eg."he*lo") is not supported.
```

Comment:
This only says the peculiarity of xapian's wildcards, not explaining exactly what wildard that GD supports.